libs/ui/switch/helm/jest.config.ts

```diff
@@ -14,6 +14,7 @@ export default {
 		],
 	},
 	transformIgnorePatterns: ['node_modules/(?!.*\\.mjs$)'],
+	testPathIgnorePatterns: ['<rootDir>/src/lib/hlm-switch.ng-model.component.spec.ts'],
```

Comment:
Can we use something that is more obvious in the test file, e.g. `test.skip`? I could see this be a source of confusion on why tests are not being executed. Again, not a blocker!

Reply:
yes I renamed it to ignore since it is added to IgnorePatterns. 
also changed this pattern in brn.